src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -381,6 +382,7 @@ ReportingConfigForm.defaultProps = {
 };
 
 ReportingConfigForm.propTypes = {
+  enterpriseId: PropTypes.string.isRequired,
```

Comment:
nit: I think we should rename the prop to `enterpriseCustomerId` to make it consistent with other places?